components/boards/AllArticleItem.tsx

```diff
@@ -2,20 +2,21 @@ import { ArticleProp } from "@/types/articleTypes";
 import styles from "./Freeboard.module.scss";
 import favoriteImg from "@/public/images/icons/ic_heart.svg";
 import { getFormatTime } from "@/utils/Utils";
+import noImg from "@/public/images/icons/no_img.svg";
 
 export default function AllArticleItem({
   article: { createdAt, image, likeCount, title, writer },
 }: ArticleProp) {
+  const titleImage = image ? image : noImg.src;
```

Comment:
이미지가 true 라면 그대로 image를 할당하면 되므로 `||` 연산자를 활용해보시면 좋을 것 같습니다. 😁

```suggestion
  const titleImage = image || noImg.src;
```

Reply:
|| 연산자를 쓰면 훨씬 간단하네요. 삼항연산자만 너무 자주 애용해서😂 다른 연산자도 생각해보겠습니다~!!🫡